test/test-v8debugapi.ts

```diff
@@ -723,9 +751,11 @@ describe('v8debugapi', function() {
             return resolved && (resolved.status as any).isError;
           }));
 
-          api.clear(bp);
           config.capture.maxDataSize = oldMaxData;
```

Comment:
The same concern about moving this line inside the callback of `api.clear` as described above.